<launch>
  <!-- segmentation parameters -->
  <rosparam param="parse/scene/min_x"> 0.2 </rosparam>
  <rosparam param="parse/scene/max_x"> 1.2 </rosparam>
  <rosparam param="parse/scene/min_y"> -1 </rosparam>
  <rosparam param="parse/scene/max_y"> 1 </rosparam>
  <rosparam param="parse/scene/min_z"> 0.3 </rosparam>
  <rosparam param="parse/scene/max_z"> 1.7 </rosparam>
  <rosparam param="parse/hsurface/distance_threshold"> 0.015 </rosparam>
  <rosparam param="parse/hsurface/eps_angle"> 0.087266463 </rosparam>
  <!-- 5 degrees is 0.087266463 radians -->
  <rosparam param="parse/objects/distance_threshold"> 0.05 </rosparam>
  <rosparam param="parse/objects/point_color_threshold"> 35 </rosparam>
  <rosparam param="parse/objects/region_color_threshold"> 20 </rosparam>
  <rosparam param="parse/objects/min_cluster_size"> 38 </rosparam>

  <node pkg="tangible" type="scene_visualization_main" name="scene_visualization">
    <remap from="cloud_in" to="/camera/depth_registered/points" />
  </node>
</launch>
